deps/jerry/jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-uint8array-prototype.c

```diff
@@ -40,4 +54,131 @@
  * @}
  */
 
+/**
+ * The Uint8Array.prototype object's 'slice' routine
+ *
+ * See also:
+ *          ECMA-262 v5, 15.4.4.10
```

Comment:
See https://www.ecma-international.org/ecma-262/6.0/#sec-%typedarray%.prototype.slice, this should be `ECMA-262 v6, 22.2.3.23`.

Reply:
> > The interpretation and use of the arguments of %TypedArray%.prototype.slice are the same as for Array.prototype.slice as defined in 22.1.3.22.
> 
> As the ecma said, the Array and TypedArray share the same algorithm, thus I think we could use the same function to simplify their implementation.

I've tried multiplex the implement of Array, but it may not distinct, so I copy it to typedarray.